### PR TITLE
implemented 2 target distance

### PR DESCRIPTION
**Pull request by JorisHeemskerk**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -71,7 +72,14 @@
...
 
-teams = [team1]
+team3 = TwoTargetsDistance(
+    copy.deepcopy(targetscoords),
+    2,
+    settings.PLANE_POLIKARPOV_I_16,
+    0
+)
+
+teams = [team3]
```
Comment (JorisHeemskerk) [inline main.py:82]: team3?

File: two_targets_distance_team.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        self, 
+        targets: list[tuple[float, float]],
+        n_agents: int,
+        agent_description: dict,
+        team_number: int
+    ) -> None:
+        """
+        Constructor for AbsoluteDistanceTeam.
+        Creates member variables using super Team class.
+        Assigns starting targets to agents
```
Comment (JorisHeemskerk) [inline two_targets_distance_team.py:27]: update docstring

File: two_targets_distance_team.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            agent : Agent, 
+            targets :list[tuple[float,float]]
+        ) -> float:
+        """
+        This function bets on a target by calculating the shortest distance 
+        between the agent and two targets.
+
+        :param target1: target to be bet on (target.Target)
+        :param agent: agent that bets on the target (agent.Agent)
+        :param targets: list of targets (list[target.Target])
```
Comment (JorisHeemskerk) [inline two_targets_distance_team.py:93]: target1 is a terrible variable name to pass to a function

File: two_targets_distance_team.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+            )
+        dy = abs(coords2[1] - coords2[1])
+        return np.sqrt(dx**2 + dy**2)
+        
+
+    def _calculate_path(
+            self, 
+            target1 : tuple[float, float], 
+            agent : Agent, 
+            targets :list[tuple[float,float]]
```
Comment (JorisHeemskerk) [inline two_targets_distance_team.py:85]:
```suggestion
            targets :list[tuple[float, float]]
```

File: two_targets_distance_team.py [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            return None #change to whatever needed to show that all targets are gone
+
+        for target2 in targets:
+            if target1[0] != target2[0] or target1[1] != target2[1]:
+                distance = self._calculate_distance(target1, target2)
+                total_distance = agent_to_target_distance + distance 
+                distance_dict[total_distance] = (target1, target2)    
+                #this is dict is unused atm, but if we want to expand
+                # the function so it can tell the other agent to fuck
+                # off, we can **expand**
```
Comment (JorisHeemskerk) [inline two_targets_distance_team.py:112]: not pep, but also not very nice english

File: two_targets_distance_team.py [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+        :return: the shortest distance between the agent and two targets.
+        """
+        distance_dict = {}
+        agent_pos = agent.rot_rect.center 
+        agent_to_target_distance = self._calculate_distance(agent_pos, target1)
+        if len(targets) == 1:
+            return agent_to_target_distance
+        if len(targets) == 0:
+            return None #change to whatever needed to show that all targets are gone
```
Comment (JorisHeemskerk) [inline two_targets_distance_team.py:103]: leftover todo

File: two_targets_distance_team.py [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+                #this is dict is unused atm, but if we want to expand
+                # the function so it can tell the other agent to fuck
+                # off, we can **expand**
+        return min(distance_dict.keys())
+
+    def assign_targets(self) -> None:
+        """
+        Bidding function for team. 
+        This function uses the absolute distance from an agent to a target
+        to bid for one.
```
Comment (JorisHeemskerk) [inline two_targets_distance_team.py:119]: incorrect docstring

File: two_targets_distance_team.py [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+            agent_description,
+            team_number
+        )
+        
+        self.assign_targets()
+    
+    def _calculate_distance(
+        self, 
+        coords1: tuple[float, float], 
+        coords2: tuple[float, float]
```
Comment (JorisHeemskerk) [inline two_targets_distance_team.py:62]: not conform with docstring, and also terrible variable names

**Review by Timbaken — APPROVED** [review]

**Pull request merged**